pandas/_libs/lib.pyx

```diff
@@ -693,6 +696,9 @@ cpdef ndarray[object] ensure_string_array(
         The value to use for na. For example, np.nan or pd.NA.
     convert_na_value : bool, default True
         If False, existing na values will be used unchanged in the new array.
+    coerce : bool, default True
```

Comment:
see above

Reply:
but this actually is very confusing with the convert_na_value. should not have both.